lib/models/toponyme.js

```diff
@@ -175,5 +175,5 @@ module.exports = {
   updateSchema,
   fetchOne,
   fetchAll,
-  remove
+  remove,
```

Comment:
Étonnant que le linter ne dise rien.

```suggestion
  remove
```